podspec_helpers/mixed_readiness_components.rb

```diff
@@ -0,0 +1,66 @@
+module MixedReadinessComponents
```

Comment:
Please add license preamble comments + docs explaining this file's purpose.

Reply:
Done in 9cd488d

Reply:
And e476a2c